src/MercadoPago.Tests/Client/Mock/CardPaymentWith3dsRequest.json

```
@@ -64,6 +64,5 @@
   "token": "bf9edf6ffae3ab5742033f33c557d54e",
   "transaction_amount": 100,
   "statement_descriptor": "statementDescriptor",
-  },
-  "three_d_secure_mode":  "optional"
-}
+  "three_d_secure_mode": "optional"
```

Não está faltando um "s" aqui?
```suggestion
  "three_ds_secure_mode": "optional"
```

---

Não @eltinMeli , o "s" é de secure e o atributo é dessa maneira mesmo para criação do pagamento. Pode ser verificado [nesse arquivo](https://docs.google.com/document/d/1QS4LfDRKpseI7OFofe66vltILaZkJ8Npg-rBqQvqjRY/edit) que disponibilizaram para nós com exemplos de request e response para o 3DS.